maquetado/scss/description.scss

```diff
@@ -0,0 +1,119 @@
+@import "colors";
+
+.description-line-separator {
+  position: relative;
+  &::after {
+    background-color: $green;
+    content: "";
+    height: 5px;
+    margin-top: 20px;
+    position: absolute;
+    width: 640px;
+    top: calc(70%);
+  }
+}
+
+.description-image-container {
+  position: relative;
+  &::before {
+    content: "";
+    background: url("../maquetado/image/badge.png") no-repeat center/contain;
+    height: 100px;
+    left: 200px;
+    position: absolute;
+    transform: rotate(15deg);
+    width: 100px;
+  }
+}
+
+.description-book-cover {
+  background-color: $white;
+  box-shadow: 5px 5px 10px 0 $black-05;
+  display: flex;
+  height: 415px;
+  margin-left: 50px;
+  margin-top: 60px;
+  max-width: 1000px;
+  padding: 10px;
+  width: 100%;
+}
+
+.description-book-image {
+  height: 370px;
+  margin-left: 10px;
+  margin-top: 10px;
+  width: 260px;
+}
+
+.description-book-content-title {
+  display: flex;
+  flex-direction: column;
+  margin-left: 20px;
+}
+
+.description-book-genre {
+  color: $grey;
+  font-weight: bold;
+  font-size: 30px;
+  line-height: 40px;
+}
+
+.description-book-detail {
+  display: flex;
+  margin-left: 10px;
+  margin-top: 20px;
+  margin-bottom: 14px;
+}
+
+.description-book-title {
+  font-family: Roboto;
```

Comment:
I think its better to have a fall-back font, so you can change `font-family: Roboto;` for `font-family: Roboto, Helvetica, Arial, sans-serif;`
An in all `font-family` uses.